tao/tools/download_protoc.sh

```diff
@@ -25,7 +25,7 @@ mkdir -p ${download_dir}
 protoc_dir="protoc-${pb_ver}-linux-x86_64"
 protoc_file="${protoc_dir}.zip"
 protoc_url="http://gitlab.alibaba-inc.com/odps_tensorflow/other/raw/master/github.com/protocolbuffers/protobuf/releases/download/v${pb_ver}/${protoc_file}"
-protoc_url_bak="https://github.com/protocolbuffers/protobuf/releases/download/v${pb_ver}/${protoc_file}"
+protoc_url_bak="https://blade-disc-remote-cache.oss-cn-hongkong.aliyuncs.com/download/protobuf/v${pb_ver}/${protoc_file}"
```

Comment:
This bucket is for build cache in my understanding from it's name ... 
Another, the URL above `gitlab.alibaba-inc.com` should be removed in this repo?

Reply:
update the bucket to `bladedisc-ci `.

> This bucket is for build cache in my understanding from it's name ...
Another, the URL above gitlab.alibaba-inc.com should be removed in this repo?

Agree with that, and I keep the github URL as the backup.